packages/Testing/PHPUnit/AbstractTestCase.php

```diff
@@ -29,7 +30,8 @@ protected function boot(): void
      */
     protected function bootFromConfigFiles(array $configFiles): void
     {
-        $configsHash = $this->createConfigsHash($configFiles);
+        $fileHasher = new FileHasher();
```

Comment:
I think it could be autowired with the `#[Required]` attribute, it's something already done over the code